doc/custom-build.md

```diff
@@ -66,7 +66,7 @@ Configuring pre-defined devices will allow the service to automatically provisio
 
 
 ### Configure the Device Service
-1. Open the [configuration.toml](./cmd/res/configuration.toml) file using your preferred text editor
+1. Open the [configuration.yaml](./cmd/res/configuration.yaml) file using your preferred text editor
 
 1. Make sure `secret name` is set to match `SecretName` in `camera.toml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
```

Comment:
```suggestion
1. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
```

Reply:
@jim-wang-intel , actual those are not in scope. There is a separate SDK PR to move those files to YAML which when consumed should make those changes.

Reply:
not sure what you meant, this is md docs

Reply:
@vli11 , make sure all TOML examples are converted to YAML.